src/main/scala/gpp/ui/reusability/package.scala

```diff
@@ -20,13 +19,6 @@ trait UtilReusabilityInstances {
     Reusability.by(z => (z.lefts, z.focus, z.rights))
 }
 
-/**
-  * Instances of reusability for some model types
-  */
-trait ModelReusabilityInstances {
-  implicit val obsIdReuse: Reusability[Observation.Id] = Reusability.by(_.format)
-}
-
```

Comment:
I removed this since we don't have `Observation` for the moment. (In explore we are temporarily using an `ExploreObservation`)